app/docker/components/datatables/containers-datatable/containersDatatable.html

```diff
@@ -9,6 +9,57 @@
           <span class="setting" ng-class="{ 'setting-active': $ctrl.state.displayTextFilter }" ng-click="$ctrl.updateDisplayTextFilter()" ng-if="$ctrl.showTextFilter">
             <i class="fa fa-search" aria-hidden="true"></i> Search
           </span>
+          <span class="setting" ng-class="{ 'setting-active': $ctrl.columnVisibility.state.open }" uib-dropdown dropdown-append-to-body auto-close="disabled" is-open="$ctrl.columnVisibility.state.open">
```

Comment:
Could you rebase your PR on the latest develop branch? It will be easier to review if I can only focus on the changes related to this PR.

Reply:
True. My bad.